src/pages/verify/e-ids/german-personalausweis.mdx

```diff
@@ -0,0 +1,127 @@
+---
+product: verify
+category: eIDs
+sort: 1
+title: German Personalausweis
+subtitle: Learn more about German Personalausweis token contents, how to test your integration and how to gain access to production.
+---
+
+import GermanPersonalausweisJwtSnippet from '../../../snippets/jwt-payloads/german-personalausweis';
+
+## JWT/Token example
+
+Triggered with `acr_values=urn:grn:authn:de:personalausweis`.
+
+<Highlight icon="info">
+
+The level of assurance for all Personalausweis authentications is: High.
+[Learn more about assurance levels](https://ec.europa.eu/digital-building-blocks/wikis/display/DIGITAL/eIDAS+Levels+of+Assurance).
```

Comment:
Should external links open in a new tab? (I usually do this, but I see we're not always consistent throughout the docs.)

Reply:
New tab or not cannot be handled at the mdx level.
But yes we probably should (can likely be fixed via MdxProvider)

Reply:
No `<a href="..." target="_blank">` @mickhansen ?

Reply:
@nmoskaleva That would work, but easier to just check if it's an external link via MdxProvider

Reply:
@fkj Correct, needs a general fix.